datasets/has_part/README.md

```diff
@@ -0,0 +1,175 @@
+  - [Table of Contents](#table-of-contents)
+  - [Dataset Description](#dataset-description)
+    - [Dataset Summary](#dataset-summary)
+    - [Supported Tasks and Leaderboards](#supported-tasks-and-leaderboards)
+    - [Languages](#languages)
+  - [Dataset Structure](#dataset-structure)
+    - [Data Instances](#data-instances)
+    - [Data Fields](#data-fields)
+    - [Data Splits](#data-splits)
+  - [Dataset Creation](#dataset-creation)
+    - [Curation Rationale](#curation-rationale)
+    - [Source Data](#source-data)
+      - [Initial Data Collection and Normalization](#initial-data-collection-and-normalization)
+      - [Who are the source language producers?](#who-are-the-source-language-producers)
+    - [Annotations](#annotations)
+      - [Annotation process](#annotation-process)
+      - [Who are the annotators?](#who-are-the-annotators)
+    - [Personal and Sensitive Information](#personal-and-sensitive-information)
+  - [Considerations for Using the Data](#considerations-for-using-the-data)
+    - [Social Impact of Dataset](#social-impact-of-dataset)
+    - [Discussion of Biases](#discussion-of-biases)
+    - [Other Known Limitations](#other-known-limitations)
+  - [Additional Information](#additional-information)
+    - [Dataset Curators](#dataset-curators)
+    - [Licensing Information](#licensing-information)
+    - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://allenai.org/data/haspartkb
+- **Repository:**
+- **Paper:** https://arxiv.org/abs/2006.07510
+- **Leaderboard:**
+- **Point of Contact:** Peter Clark <peterc@allenai.org>
+
+### Dataset Summary
+
+This dataset is a new knowledge-base (KB) of hasPart relationships, extracted from a large corpus of generic statements. Complementary to other resources available, it is the first which is all three of: accurate (90% precision), salient (covers relationships a person may mention), and has high coverage of common terms (approximated as within a 10 year old’s vocabulary), as well as having several times more hasPart entries than in the popular ontologies ConceptNet and WordNet. In addition, it contains information about quantifiers, argument modifiers, and links the entities to appropriate concepts in Wikipedia and WordNet.
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+English
+
+## Dataset Structure
+
+### Data Instances
+
+[More Information Needed]
+
+### Data Fields
+
+```json
+{
+  "arg1": {
+    "feature_type": "Value",
+    "dtype": "string"
+  },
+  "arg2": {
+    "feature_type": "Value",
+    "dtype": "string"
+  },
+  "score": {
+    "feature_type": "Value",
+    "dtype": "float64"
+  },
+  "wikipedia_primary_page": {
+    "feature_type": "Sequence",
+    "feature": {
+      "feature_type": "Value",
+      "dtype": "string"
+    }
+  },
+  "synset": {
+    "feature_type": "Sequence",
+    "feature": {
+      "feature_type": "Value",
+      "dtype": "string"
+    }
+  }
+}
+```
```

Comment:
Here we expect a description of the fields in natural language.
Can you add a brief description for each field please ?

Reply:
Can you add a description of each field `arg1` `arg2` `score` `wikipedia_primary_page` `synset` ?